pkg/runner/testdata/uses-composite/composite_action/action.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: "Test Composite Action"
+description: "Test action uses composite"
+
+inputs:
+  test_input_required:
+    description: "Required input"
+    required: true
+  test_input_optional:
+    description: "optional defaulted input"
+    required: false
+    default: "test_input_optional_value"
+
+outputs:
+  test_output:
+    description: "Output value to pass up"
+    value: ${{ step.output.outputs.test_output }}
+
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        echo "#####################################"
+        echo "Inputs:"
+        echo "---"
+        echo "test_input_required=${{ inputs.test_input_required }}"
+        echo "test_input_optional=${{ inputs.test_input_optional }}"
+        echo "---"
+
+    # Let's test the inputs
+    - run: |
+        if [ "${{ inputs.test_input_required }}" != "test_input_required_value" ]; then
+          exit 1
+        fi
+
+    - run: |
+        if [ "${{ inputs.test_input_optional }}" != "test_input_optional_value" ]; then
+          exit 1
+        fi
+
+    # Let's send up an output to test
+    - run: echo "::set-output name=test_output::test_output_value"
+
+    # Composite using a 3rd party action
+    - uses: actions/checkout@v2
+
+    # Composite using a docker URL
+    - uses: docker://alpine:3.13
+      with:
+        somekey: somevalue
+        args: echo ${INPUT_SOMEKEY} | grep somevalue
```

Comment:
I don't think this should actually work, composite actions only support using `run` with an explicit `shell` (see docs [here](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsrun))

Reply:
You're absolutely right. I added these right before find this out in the real world. I've now removed those steps.